frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -123,7 +123,10 @@ export default function PersonalInformationIndex() {
           )}
         </DescriptionListItem>
         <DescriptionListItem term={t('personal-information:index.phone-number')}>
-          <p>{personalInformation.primaryTelephoneNumber}</p>
+          <dt className="font-semibold">{t('personal-information:index.primary-phone-number')}</dt>
```

Comment:
Not sure if `<dt>` should be used here since all `children` of `<DescriptionListItem>` are wrapped in a `<dd>` tag.

I'm thinking of using `<p>` or `<span>`

Reply:
I had to run this page through an HTML validator and `<dl>` (not `<dt>` though) are allowed to be nested in a `<dd>` (the element used for `children` of `<DescriptionListItem>`)

Reply:
should be fixed now after applying the code suggested on the 3rd comment